### PR TITLE
[add] additional meta tags for SEO

### PR DESCRIPTION
**Pull request by iSumitBanik**
This PR provides the addition of extra meta tags that are responsible for the SEO.

Issue #212

### CODE REVIEW

File: frontend/src/components/router-head/router-head.tsx [hunk 1]
```diff
@@ -34,6 +34,19 @@ export const RouterHead = component$(() => {
       <meta name="twitter:site" content="@QwikDev" />
       <meta name="twitter:title" content="Qwik" />
```
Comment (origranot) [inline frontend/src/components/router-head/router-head.tsx:35]: Please remove those meta tags.

File: frontend/src/components/router-head/router-head.tsx [hunk 2]
```diff
@@ -34,6 +34,19 @@ export const RouterHead = component$(() => {
...
+      <meta property="og:type" content="website" />
+      <meta property="og:title" content="Reduced.to | URL Shortener" />
+      <meta property="og:description" content="Reduced.to is a Free Modern URL Reducer" />
+      <meta property="og:url" content="https://reduced.to/" />
+      <meta property="og:site_name" content="Reduced.to" />
+      <meta name="twitter:card" content="summary_large_image" />
+      <meta name="twitter:title" content="Reduced.to | URL Shortener" />
+      <meta name="twitter:description" content="Reduced.to is a Free Modern URL Reducer" />
+      <meta name="theme-color" content="#272935" />
+
```
Comment (origranot) [inline frontend/src/components/router-head/router-head.tsx:49]: Please remove the theme-color meta tag

**Review by origranot — CHANGES_REQUESTED** [review]
@iSumitBanik 
Hey, please fix the comments 💯

File: frontend/src/components/router-head/router-head.tsx (hunk 2 above)
Comment (iSumitBanik) [reply frontend/src/components/router-head/router-head.tsx]: Done

**Comment by origranot** [comment]
@iSumitBanik We already merged a PR regarding our SEO.

Thank you so much!

**Pull request closed without merging**